lesson01/Assignment 02 - Oleg/index.html

```diff
@@ -0,0 +1,56 @@
+<html lang="en">
```

Comment:
Ensure that you add `<!DOCTYPE html>` as the first line to make this 100% semantically correct.